### PR TITLE
feat: GEOT-195: new post component

### PR DESCRIPTION
**Pull request by ra8ga**
;-),  ;-),  ;-)

### CODE REVIEW

File: packages/app-andrew/src/pages/group/index.tsx [hunk 1]
```diff
@@ -36,12 +37,13 @@ const GroupPage: NextPage<{}> = () => {
               </Button>
             </div>
           </div>
-          <div
+          {/* @todo this needs to be done better */}
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/index.tsx:40]: remove comments

File: packages/ui/PostCreate/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <div className={styles.avatar}>
+          <UserAvatar
+            image="https://www.bootdey.com/img/Content/avatar/avatar6.png"
+            additionalStyles={styles.userAvatar}
+          />
+          <span>John Lasseter</span>
+        </div>
+        <ExpandingTextarea
+          rows={3}
+          // additionalStyles={styles.text}
```
Comment (mariusz-kabala) [inline packages/ui/PostCreate/index.tsx:29]: remove comment

File: packages/ui/PostCreate/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          placeholder={'Your message...'}
```
Comment (mariusz-kabala) [inline packages/ui/PostCreate/index.tsx:30]: use translator

File: packages/ui/PostCreate/index.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        />
+      </div>
+      <Button type="submit" disabled={false} additionalStyles={styles.btn}>
+        send
```
Comment (mariusz-kabala) [inline packages/ui/PostCreate/index.tsx:34]: use translator

File: packages/app-andrew/src/pages/group/index.tsx (hunk 1 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/index.tsx]: fixed

File: packages/ui/PostCreate/index.tsx (hunk 2 above)
Comment (ra8ga) [reply packages/ui/PostCreate/index.tsx]: fixed

File: packages/ui/PostCreate/index.tsx (hunk 3 above)
Comment (ra8ga) [reply packages/ui/PostCreate/index.tsx]: fixed

File: packages/ui/PostCreate/index.tsx (hunk 4 above)
Comment (ra8ga) [reply packages/ui/PostCreate/index.tsx]: fixed

File: packages/ui/PostCreate/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    >
+      <div className={styles.text}>
+        <div className={styles.avatar}>
+          <UserAvatar
+            image="https://www.bootdey.com/img/Content/avatar/avatar6.png"
+            additionalStyles={styles.userAvatar}
+          />
+          <span>John Lasseter</span>
+        </div>
+        <ExpandingTextarea rows={3} placeholder={t('your.message')} />
```
Comment (mariusz-kabala) [inline packages/ui/PostCreate/index.tsx:27]: dot is nesting the keys, use here `your-message`

Comment (mariusz-kabala) [reply packages/ui/PostCreate/index.tsx:27]: and add new *keys* to english json

Comment (ra8ga) [reply packages/ui/PostCreate/index.tsx]: fixed

File: packages/app-andrew/src/pages/group/index.tsx [hunk 6]
```diff
@@ -68,6 +63,7 @@ const GroupPage: NextPage<{}> = () => {
           </div>
           <div className={styles.column}>
             <h2 className={styles.header}>Ostatnio dodane posty</h2>
+            <PostCreate additionalStyles={styles.postCreate} />
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/index.tsx:66]: you did not add PostCreate translation namespace to the page

Comment (ra8ga) [reply packages/app-andrew/src/pages/group/index.tsx:66]: fixed

**Pull request merged**